appinfo/info.xml

```diff
@@ -8,7 +8,7 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.2" />
 	</dependencies>
-	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
+	<description>Calendar GUI for the integrated CalDAV server</description>
```

Comment:
Integrated into what? - "for the new server-integrated CalDAV-Backend describes it better I think :wink: